### DISCUSSION TITLE
Busted Faces - GFPGAN ?

### DISCUSSION THREAD

**Discussion by swamp-cabbage** [lllyasviel/Fooocus#2693] [Q&A] [answered]
I have been generating a lot of "busted faces" - what SD spits out when the subject is too far from the viewer, not enough data, etc. In SD.Next I use the "face restore" function with GFPGAN or Codeformer. 

Do we have any options to use in Fooocus which have a similar effect ?

Thanks !

**Comment by ILikeToasters** [comment, marked as answer]
In the inpainting section you will find the improve detail option which is good for that. Mask the face and choose that option and in the extra prompt area to the left put something like "woman's detailed face". Works pretty well overall.
<img width="467" alt="Screenshot 2024-04-01 160122" src="https://github.com/lllyasviel/Fooocus/assets/4357657/b2812f00-ecaa-4501-85d4-e799dd03b61d">

**Reply by swamp-cabbage** [reply]
Thanks !
